.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: CI
+on:
+  push:
```

I think adding `pull_request` would also be useful.
```suggestion
  push:
  pull_request:
```

---

Done @straight-shoota. Thanks.